public/components/common/helpers/ppl_docs/commands/dedup.ts

```diff
@@ -109,6 +88,6 @@ PPL query:
     | 18               | M        |
     +------------------+----------+
 
-#### Limitation
-The \`dedup\` command is not rewritten to OpenSearch DSL, it is only executed on the coordination node.
+### Limitation
```

Comment:
Does this limitation still exist? What is OpenSearch DSL? Do you mean Query DSL?

Reply:
Yes, true b/c the two "duplicate" records could be assigned to separate nodes b/c of the way sharding is split across nodes.  So any one node doesn't have enough information (sometimes) to correctly find a duplicate.
Only the coordinating node, which eventually receives the entire "result" can then dedup records.